src/main/scala/ru/org/codingteam/horta/protocol/jabber/MucParticipantStatusListener.scala

```diff
@@ -5,11 +5,26 @@ import akka.actor.ActorRef
 import ru.org.codingteam.horta.messages._
 
 class MucParticipantStatusListener(muc: MultiUserChat, room: ActorRef) extends DefaultParticipantStatusListener {
+
+  // TODO: Test for changing the user status from visitor to participant and back.
```

Comment:
Should we create an issue for this? You know, with `refactoring` tag or something like that.